css/style.css

```diff
@@ -0,0 +1,15 @@
+body {
+  text-align: center;
+  font-size: 20px;
+}
+form {
+  display: inline-block;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 20px 150px;
+  background-color: rgba(128, 128, 128, 0.584);
+  text-align: left;
+}
+form * {
+  margin: 8px 0px;
+}
```

Comment:
Please fix linters errors to follow the style guidelines that shared by all Microverse student

Reply:
**[OPTIONAL]:** For a better understanding of the requested change, I would recommend including a link to the guidelines or mentioned information in the comment. In this way, the student can easily grasp the idea behind the implementation to be made :heavy_check_mark: